baybe/searchspace/discrete.py

```diff
@@ -230,6 +231,99 @@ def discrete_parameter_factory(
 
         return cls(parameters=parameters, exp_rep=df, empty_encoding=empty_encoding)
 
+    @classmethod
+    def from_simplex(
+        cls,
+        parameters: List[NumericalDiscreteParameter],
+        total: float,
```

Comment:
@AVHopp @Scienfitz: what do you think about the names of the kw-args `total` and `boundary_only`. Could not come up with better ideas, perhaps you have some.... Especially `total` could be a bit confusing, because if `boundary_only=False`, the parameter sum can also be smaller that `total`.

Reply:
fine with both but indeed the `max` version might be better due to the mentioned case

Reply:
Problem with `max` is that it's a python keyword. So `max_sum` then?

Reply:
(yes that is what I meant just like alex)

Reply:
Alright, fixed